<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_update">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QTextBrowser" name="n3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>191</width>
       <height>221</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;                            &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;Main用于接收最新的稳定版本                             &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;Monthly用于接收较稳定的Beta更新，新加入的功能可能会因为bug用不了，如果您新发现了Bug请前往Github提交issues                                                        &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;Dev用于接收最新开发版，但Bug不是一般的多，如果您不是想协助我修复Bug请不要选择此通道(推荐先issues讨论然后再pull                            request)&lt;/span&gt;                         &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_update">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>Main</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Monthly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dev</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="n2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>更新通道:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="dl_update">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>检查MSL2更新</string>
     </property>
    </widget>
    <widget class="QLabel" name="n5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>说明：</string>
     </property>
    </widget>
    <widget class="QLabel" name="n4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>更新日志</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="update_log">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="n1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>更新设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_download">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QPushButton" name="pbtn_ufl2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>常用插件列表2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_download_website_manage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>下载网页管理插件（需要服务端支持插件）</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_ufl1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>常用插件列表1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_download_auto_backup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>下载自动备份插件（需要服务端支持插件）</string>
     </property>
    </widget>
    <widget class="QLabel" name="n9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>以下插件我觉得特别有用但都不是我写的:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_goto">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>前往</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_goto_website">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>Forge官网</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fabric官网</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cbukkit官网(下载)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spigot官网(下载)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vanilla官网(下载)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="n7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_view_at_docment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>查看文档中的Java安装方法</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_how_to_choice">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>如何选择</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbtn_download">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
    <widget class="QLabel" name="n6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>通过apt下载Java（不推荐，仅Ubuntu等少数系统可用）:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_want_to_download">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>87</width>
       <height>26</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <item>
      <property name="text">
       <string>Java17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Java16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Java8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Java7</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pbtn_goto_mosl">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>前往moslauncher.tk下载</string>
     </property>
    </widget>
    <widget class="QLabel" name="n8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>250</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Win/macOS可以选择：</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
